2010-08-07  Miloslav Trmaƒç  <mitr@redhat.com>

	* tmpwatch.c (usage): Add missing space.

	* tmpwatch.8: Add -X to SYNOPSIS as well.

	* tmpwatch.c (usage): Mention -X, --exclude-pattern.
	* tmpwatch.8: Add --exclude-pattern to SYNOPSIS.

diff -r 7dca472557ba tmpwatch.8
--- a/tmpwatch.8	Fri Jul 23 18:27:36 2010 +0200
+++ b/tmpwatch.8	Sat Aug 07 05:55:07 2010 +0200
@@ -26,10 +26,11 @@
 .SH NAME
 tmpwatch \- removes files which haven't been accessed for a period of time
 .SH SYNOPSIS
-\fBtmpwatch\fR [-u|-m|-c] [-MUadfqstvx] [--verbose] [--force] [--all]
+\fBtmpwatch\fR [-u|-m|-c] [-MUXadfqstvx] [--verbose] [--force] [--all]
                [--nodirs] [--nosymlinks] [--test] [--fuser] [--quiet]
                [--atime|--mtime|--ctime] [--dirmtime] [--exclude \fIpath\fR]
-               [--exclude-user \fIuser\fR] \fItime\fR \fIdirs\fR
+               [--exclude-user \fIuser\fR] [--exclude-pattern \fIpattern\fR]
+               \fItime\fR \fIdirs\fR
 
 .SH DESCRIPTION
 \fBtmpwatch\fR recursively removes files which haven't been accessed
diff -r 7dca472557ba tmpwatch.c
--- a/tmpwatch.c	Fri Jul 23 18:27:36 2010 +0200
+++ b/tmpwatch.c	Sat Aug 07 05:55:07 2010 +0200
@@ -600,10 +600,10 @@
 static void attribute__((noreturn))
 usage(void)
 {
-    static const char msg[] = "tmpwatch [-u|-m|-c] [-MUadfqtvx] [--verbose] "
+    static const char msg[] = "tmpwatch [-u|-m|-c] [-MUXadfqtvx] [--verbose] "
 	"[--force] [--all] [--nodirs] [--nosymlinks] [--test] [--quiet] "
 	"[--atime|--mtime|--ctime] [--dirmtime] [--exclude <path>] "
-	"[--exclude-user <user>] "
+	"[--exclude-user <user>] [--exclude-pattern <pattern>] "
 #ifdef FUSER
 	"[--fuser] "
 #endif
